openerp/addons/test_impex/tests/test_export.py

```diff
@@ -23,7 +23,7 @@ def make(self, value):
 
     def export(self, value, fields=('value',), context=None):
         record = self.make(value)
-        return record._BaseModel__export_rows([f.split('/') for f in fields])
+        return record._BaseModel_export_rows([f.split('/') for f in fields])
```

Comment:
@sebastienbeau no need for name mangling anymore: `_BaseModel__export_rows` becomes `_export_rows`.